google-cloud-bigquery/lib/google/cloud/bigquery/convert.rb

```diff
@@ -56,7 +56,7 @@ def self.format_rows rows, fields
         ##
         # @private
         def self.format_row row, fields
-          row_pairs = fields.zip(row[:f]).map do |f, v|
+          row_pairs = fields.zip(row["f"]).map do |f, v|
```

Comment:
I'm a little nervous about type conversions (nested structs particularly), but this seems to pass all tests. Are reviewers comfortable with this?

Reply:
I'm confident in our test coverage, so if it passes I have no problem with the change.

Reply:
The more I think about this, the more I would prefer that we keep the hash keys as symbols and not convert to use strings. You can control this with the [`symbolize_names`](http://ruby-doc.org/stdlib-2.4.2/libdoc/json/rdoc/JSON.html#method-i-parse) argument.

Reply:
I didn't know this was possible, thank you for pointing out. Done.